run.sh failure due to yarn

We are seeing run.sh fail with these yarn errors -- any idea what's going on?:

```
    ERROR in ./galaxy/style/scss/base.scss (./node_modules/css-loader/dist/cjs.js??ref--8-1!./node_modules/postcss-loader/src??ref--8-2!./node_      modules/sass-loader/lib/loader.js??ref--8-3!./galaxy/style/scss/base.scss)
    Module build failed (from ./node_modules/sass-loader/lib/loader.js):

    @import "../../../node_modules/@fortawesome/fontawesome-free/scss/fontawesome";
    ^
          File to import not found or unreadable: ../../../node_modules/@fortawesome/fontawesome-free/scss/fontawesome.
          in /data/galaxy/client/galaxy/style/scss/base.scss (line 15, column 1)
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js??ref--8-1!node_modules/vue-loader/lib/loaders/stylePostLoader.js!node_modules      /postcss-loader/src/index.js??ref--8-2!node_modules/sass-loader/lib/loader.js??ref--8-3!node_modules/vue-loader/lib/index.js??vue-loader-option      s!galaxy/scripts/components/Markdown/Markdown.vue?vue&type=style&index=0&lang=scss&:
    Entrypoint mini-css-extract-plugin = *


...
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Thanks for helping.  We have been tweaking the client styling and somehow we got in a state where there's mismatches in some of the styling files and what's installed by node I think.  We rolled back the styling changes in base.scss, and I had already done a yarn update, and that cleared some of the errors...but we are still left with some client build errors.  Here's one of them....can you point us in the direction of how to fix this?

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/command-aliases' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 6:9-49
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

OK, I did that.... the same set of errors still persist after yarn run build is executed:

```
ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/command-aliases' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 6:9-49
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/convert' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 7:9-41
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/latex/commands/1args/index' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 14:9-60
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/options' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 8:9-41
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/tex/KnownCommand' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 11:9-50
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/tex/KnownCommand0Args' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 12:9-55
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/tex/KnownCommand1Args' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 13:9-55
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/unknown-command' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 9:9-49
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

ERROR in ./node_modules/latex-to-unicode-converter/index.js
Module not found: Error: Can't resolve './ts-compiled/util' in '/data/galaxy/client/node_modules/latex-to-unicode-converter'
 @ ./node_modules/latex-to-unicode-converter/index.js 10:9-38
 @ ./node_modules/babel-loader/lib??ref--1!./node_modules/vue-loader/lib??vue-loader-options!./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue?vue&type=script&lang=js&
 @ ./galaxy/scripts/components/Citations.vue
 @ ./galaxy/scripts/entry/analysis/AnalysisRouter.js
 @ ./galaxy/scripts/entry/analysis/index.js
 @ multi polyfills bundleEntries entry/analysis

...
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js??ref--8-1!node_modules/vue-loader/lib/loaders/stylePostLoader.js!node_modules/postcss-loader/src/index.js??ref--8-2!node_modules/sass-loader/dist/cjs.js??ref--8-3!node_modules/vue-loader/lib/index.js??vue-loader-options!galaxy/scripts/components/User/CloudAuth/CloudAuth.vue?vue&type=style&index=0&lang=scss&:
    Entrypoint mini-css-extract-plugin = *
       9 modules
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/client/package.json
+++ b/client/package.json
@@ -36,7 +36,7 @@
     "jquery.complexify": "^0.5.2",
     "jquery.cookie": "^1.4.1",
     "latex-parser": "^0.6.2",
-    "latex-to-unicode-converter": "^0.5.2",
+    "latex-to-unicode-converter": "0.5.2",
     "markdown-it": "^8.4.2",
     "moment": "2.20.1",
     "popper.js": "^1.14.7",

